src/main/scala/me/rexim/issuestant/Main.scala

```diff
@@ -1,13 +1,32 @@
 package me.rexim.issuestant
 
+import me.rexim.issuestant.github.EventsSource
+
+import scala.concurrent.duration.{FiniteDuration, SECONDS}
+
 import scalaz.concurrent.Task
 
 import org.http4s.server.{Server, ServerApp}
 import org.http4s.server.blaze._
+import org.http4s.client.blaze._
 
 object Main extends ServerApp {
-  // TODO: Use the HTTP port passed by heroku
+  def pollingLoop(permalink: Permalink): Unit = {
+    permalink.update.flatMap { nextPermalink =>
+      Task.schedule(pollingLoop(nextPermalink), FiniteDuration(1, SECONDS))
```

Comment:
Yeah, I completely agree. I'm in the middle of doing exactly that. :D I'm also research how we can apply scalaz-streams here. Looks like a thing that can fit our purposes. Will update shortly on that.

Thanks for the remark!